pkg/controller/controller.go

```diff
@@ -632,13 +632,16 @@ func (cont *AciController) Run(stopCh <-chan struct{}) {
 			cont.opflexDeviceDeleted(dn)
 		})
 	go cont.apicConn.Run(stopCh)
-	go cont.enableGlobalSync(stopCh)
+	if !cont.config.DisablePeriodicSnatGlobalInfoSync {
+		go cont.enableGlobalSync(stopCh)
+	}
 }
 
 func (cont *AciController) enableGlobalSync(stopCh <-chan struct{}) {
 	time.Sleep(time.Minute)
+	cont.log.Info("Go routine to periodically sync globalinfo and nodeinfo started")
 	for {
-		cont.log.Debug("Running code to sync GlobalInfo with Node infos")
+		cont.log.Debug("Syncing GlobalInfo with Node infos")
```

Comment:
This is getting logged in ever loop iteration, can we log it every 5th or 10 iteration to reduce the logging noise in the debug level?

Reply:
Will take of that